Cargo.toml

```diff
@@ -20,3 +20,18 @@ wasm-bindgen-cli-support = "0.2.68"
 
 [target.'cfg(unix)'.dependencies]
 libc = "0.2"
+
+[dev-dependencies]
+anyhow = "1.0"
+cargo_metadata = "0.14"
+env_logger = "0.9"
+log = "0.4"
+structopt = { version = "0.3", default-features = false }
+wasm-bindgen-cli-support = "0.2.68"
```

Comment:
lot of duplicated dependencies in there...

if a dependency already exists in `[dependencies]`, you don't need to add it again in dev dependencies